cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,17 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://react-redux.realworld.io/#/login?_k=zczlgv');
 
+    cy.get(':nth-child(1) > .form-control')
+      .type('lurudosu@mailinator.com');
+    cy.get(':nth-child(2) > .form-control')
+      .type('Pa$$w0rd!');
+
```

Comment:
extra empty lines are not really needed - check code style here https://mate-academy.github.io/qa-program/checklists/cypress.html